### DISCUSSION TITLE
下载速度是否跟api有关?

### DISCUSSION THREAD

**Discussion by aliuzzz** [yutto-dev/yutto#263] [Q&A]
单视频下载速度最高不会超过50M/s 甚至30m/s
是api锁速度了吗

**Comment by SigureMo** [comment]
> 单视频下载速度最高不会超过50M/s 甚至30m/s

原来这算慢的吗？

可以先确认下瓶颈是否在 CPU / 磁盘读写，不然就是 API 的问题了
